.devcontainer/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Modified from https://github.com/iwahjoedi/android-devcontainer/blob/main/Image/Dockerfile
+ARG VARIENT="ubuntu-22.04"
+FROM mcr.microsoft.com/vscode/devcontainers/base:0-${VARIENT}
+
+ENV DEVCONTAINER="true"
+
+RUN apt clean && apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install git \
+    && apt-get -y install clang cmake ninja-build pkg-config \
+    && apt-get -y install wget unzip \
+    && apt-get -y install openjdk-17-jdk \
+    && apt-get clean -y \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN useradd -ms /bin/bash robolectric
+USER robolectric
+
+RUN export HOME="/home/robolectric"
+ARG HOME="/home/robolectric"
+ENV ANDROID_HOME=/home/robolectric/Android/Sdk
+ENV ANDROID_SDK_ROOT=/home/robolectric/Android/Sdk
+ENV CMDLINE_HOME="${HOME}/Android/Sdk/cmdline-tools"
+ENV CMDLINE="${HOME}/Android/Sdk/cmdline-tools/cmdline-tools/bin"
+
+# setup flutter sdk
+ENV PATH=${PATH}:${ANDROID_HOME}/platform-tools
+ENV PATH=${PATH}:${ANDROID_HOME}/platforms
+ENV PATH=${PATH}:${ANDROID_HOME}/emulators
+ENV PATH=${PATH}:${CMDLINE_HOME}:${CMDLINE}
+
+RUN ls -la ~ && whoami && mkdir -pv ${CMDLINE_HOME} && cd ${CMDLINE_HOME} \
+    && wget https://dl.google.com/android/repository/commandlinetools-linux-11076708_latest.zip \
+    && unzip commandlinetools-linux-11076708_latest.zip \
+    && rm commandlinetools-linux-11076708_latest.zip
```

Comment:
Maybe extract `11076708` is a variable, so there's only one place to change where there is an update?